iOS/UIKit Extensions/UIViewController-Extensions.swift

```diff
@@ -13,9 +13,7 @@ import Account
 extension UIViewController {
 	
 	func presentError(_ error: Error, dismiss: (() -> Void)? = nil) {
-		if let accountError = error as? AccountError, accountError.isCredentialsError {
-			presentAccountError(accountError, dismiss: dismiss)
-		} else if let decodingError = error as? DecodingError {
```

Comment:
Why was this removed?

Reply:
There are no `UIViewController`s left that make use of that code path.